pkgs/tools/bootloaders/refind/default.nix

```diff
@@ -0,0 +1,115 @@
+{ stdenv, fetchurl
+, unzip, gnu-efi, efibootmgr, dosfstools, imagemagick }:
+
+assert (stdenv.system == "x86_64-linux" ||stdenv.system == "i686-linux");
+
+stdenv.mkDerivation rec {
+
+  name = "refind-${version}";
+  srcName = "refind-src-${version}";
+  version = "0.8.4";
+
+  src = fetchurl {
+    url = "http://downloads.sourceforge.net/project/refind/${version}/${srcName}.zip";
+    sha256 = "09g6x41fc7vvwv2y9kggjmazyhdvkn8qr02v769x33f20bfiz2zc";
+  };
+
+  buildInputs = [ unzip gnu-efi efibootmgr dosfstools imagemagick ];
+
+  HOSTARCH =
+    if stdenv.system == "x86_64-linux" then "x64"
+    else if stdenv.system == "i686-linux" then "ia32" else "null";
+
+  patchPhase = ''
+    sed -e 's|-DEFI_FUNCTION_WRAPPER|-DEFI_FUNCTION_WRAPPER -maccumulate-outgoing-args|g' -i Make.common
+    sed -e 's|-DEFIX64|-DEFIX64 -maccumulate-outgoing-args|g' -i Make.common
+    sed -e 's|-m64|-maccumulate-outgoing-args -m64|g' -i filesystems/Make.gnuefi
```

Comment:
Huh, is there any specific reason why it is okay to have a `sed` call inside the patchPhase here instead of patching it in a patchfile? Just curious...